Hestia/Hestia/backend/speech_recognition/SpeechRecognition.cs

```diff
@@ -63,11 +45,12 @@ bool IsAuthorized()
             return false;
         }
 
-        public WarningMessage StartRecording()
+        public void StartRecording(out int warningStatus)
```

Comment:
I think the warningStatus is a good idea, but I find the numbering a bit confusing. 0 is apparently a Warning, while in standard C, it would indicate success. I don't know what the C# conventions are, but maybe it is an idea to use an enum?